docs/src/main/docbook/deployment.xml

```diff
@@ -193,6 +196,17 @@
         </section>
 
     </section>
+    <section  xml:id="deployment.feature.dynamicFeature">
+        <title>Feature and Dynamic Feature SPI automatic registration</title>
+        <para>
+            Since Jersey 3.1.0 there is new specification requirement to automatically discover and register classes which
```

Comment:
Please make it Since Jersey 3.1.0 there is a new specification requirement to automatically discover and register classes that implement